contracts/test/core/MockAnchor.sol

```diff
@@ -27,7 +26,7 @@ contract MockSafeCore is SafeCore {
         bytes32 _stateRoot,
         IsMemberInterface _membersManager
     )
-        SafeCore(
+        Anchor(
```

Comment:
Please update comment on L42 
`note the Core contract` -> `note the anchor contract`